src/settings/ServicePointDetails.js

```diff
@@ -50,6 +58,8 @@ class ServicePointDetails extends React.Component {
       nextPeriods: [],
       isPeriodsPending: true,
     };
+
+    moment.locale(props.intl.locale);
```

Comment:
IIRC `moment` is global and [locale is already set in stripes-core](https://github.com/folio-org/stripes-core/blob/master/src/loginServices.js#L143). Double check; you may not need to configure it here.

Reply:
I double checked. Yes it is global but unfortunately `stripes-core` does not set it that is why we have to set it in all places where `moment` is used. I believe `stripes-core` must set is but it does not. Maybe it is issue of `stripes-core` and we have to create ticket for deeper investigation because it looks like global issue for the whole application?